extensions/sensible/src/types.ts

```diff
@@ -0,0 +1,9 @@
+export type DocumentType = {
+  id: string;
+  name: string;
+  created: string;
+};
+
+export interface Preferences {
+  sensible_api_key: string;
+}
```

Comment:
```suggestion
```
This is already build into Raycast with `ExtensionPreferences`

Reply:
Oops! Forgot about that. Thanks!